pipeline/start.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+SCRIPT_DIR="$( cd "$(dirname "$0")" ; pwd -P )"
+. "${SCRIPT_DIR}/docker-run/error_handling.sh"
+. "${SCRIPT_DIR}/.local_config.sh"
+
+${ROOT_DIR}/pipeline/jobs/build_job.sh
+${ROOT_DIR}/pipeline/jobs/unittest_job.sh --testcategories BulkNormal --testdata data.zip
+${ROOT_DIR}/pipeline/jobs/unittest_job.sh --testcategories BulkNoSize --testdata data.zip
+${ROOT_DIR}/pipeline/jobs/unittest_job.sh --testcategories SVNDataLong,SVNData --testdata DSMCBE.zip
+${ROOT_DIR}/pipeline/jobs/unittest_job.sh --testcategories Border
+${ROOT_DIR}/pipeline/jobs/unittest_job.sh --testctegories Filter,Targeted,Purge,Serialization,WebApi,Utility,UriUtility,IO,ImportExport,Disruption
```

Comment:
Typo in `testctegories`?  Also, it's not clear to me how this script is used.  It looks like the test categories are duplicated here as well as in `.travis.yml`?

Reply:
Fixed it. This script is not used by travis but a script for your local environment and your convenience - it does exactly that what runs on travis, so you can execute integration tests etc from docker containers.